docs/OCP_CI_Tutorials/Scenarios/Scenarios_Guide.md

```diff
@@ -39,47 +41,57 @@ flowchart LR
   end
 
 ```
+
 ## Provision
+
 We define provision as:
 > The initial setup of your scenarios files and the OpenShift installation
 
 See the [Provision Guide](Provision_Guide.md) for detailed information meant to help you build the foundation of your scenario.
 
 ## Orchestrate
+
 We define orchestrate as:
 
 > Completing any setup or configuration necessary to execute tests. This setup can include things like: installing an operator, retrieving any necessary variables, additional configuration of the test cluster, deployment of any necessary resources on the test cluster, etc.
 
 See the [Orchestration Guide](Orchestration_Guide.md)) for detailed information meant to help you organize the installation of your product.
 
 ## Execute
+
 We define execute as:
 
 > The execution of any interop tests in a scenario. This step should only consist of any setup that absolutely must occur in this step (rather than the [orchestration step](Orchestration_Guide.md)) and the execution of interop tests. The execution of these tests should result in one or more JUnit (xml) files.
 
 See the [Execution Guide](Execution_Guide.md) for detailed information meant to help you organize your test execution.
 
 ## Report
+
 TBD
 
 See the [Reporting Guide](Reporting_Guide.md) for detailed information meant to help you report your test execution.
 
 ## Document
+
 TBD
 
 See the [Scenario Documentation Guide](../../Policy/Documentation/Scenario_Documentation_Policy.md) for detailed information meant to help you document your scenario.
 
 ## Production
+
 This is meant to be a sort of final check just to make sure that you and your reviewers are aware of what needs to be done to push to production (a.k.a merging the PR).
 
 ### Remove Spot Instances
-We cannot merge any code into the release repo meant to be used in production that is using [Spot Instances](#spot-instances). These can be unreliable and will lead to clusters going offline during production runs. We should be using them to save costs in development but when it is time to merge the scenario code we must ensure we remove the use of spot instances.
+
+We cannot merge any code into the release repo meant to be used in production that is using Spot Instances. These can be unreliable and will lead to clusters going offline during production runs. We should be using them to save costs in development but when it is time to merge the scenario code we must ensure we remove the use of spot instances.
```

Comment:
```suggestion
We cannot merge any code into the release repo meant to be used in production that is using spot instances. These can be unreliable and will lead to clusters going offline during production runs. We should be using them to save costs in development, but when it is time to merge the scenario code we must ensure we remove the use of spot instances.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)